lib/fog/vsphere/models/compute/cdroms.rb

```diff
@@ -0,0 +1,43 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdroms < Fog::Collection
+        autoload :Cdrom, File.expand_path('../cdrom', __FILE__)
```

Comment:
I guess this `autoload` needs to be inside the `Vsphere` class since it is not inside the `Cdroms` one.